docs/api/geo_datasets.csv

```diff
@@ -9,6 +9,7 @@ Dataset,Type,Source,License,Size (px),Resolution (m)
 `EnviroAtlas`_,"Imagery, Masks","NAIP, NLCD, OpenStreetMap","CC-BY-4.0",-,1
 `Esri2020`_,Masks,Sentinel-2,"CC-BY-4.0",-,10
 `EU-DEM`_,Masks,"Aster, SRTM, Russian Topomaps","CSCDA-ESA",-,25
+`EuroCrops`_,Geometries,EU Countries,"CC-BY-SA-4.0",-,-
```

Comment:
According to https://www.eurocrops.tum.de/index.html it should be:
```suggestion
`EuroCrops`_,Geometries,EU Countries,"CC-BY-4.0",-,-
```
right?

Reply:
Ah, but the GitHub and Zenodo say CC-BY-SA-4.0. That's confusing. Let's go with what you have.